### PR TITLE
[FISH-1112 FISH-1107] Payara 5.26.0 Release and TCK Docs

### PR DESCRIPTION
**Pull request by lprimak**

### CODE REVIEW

File: docs/modules/ROOT/pages/jakartaee-certification/5.26.0/tck-results-web-5.26.0.adoc [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+
+```
+[INFO] Tests run: 1043, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 632.077 s - in TestSuite
+[INFO] 
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 1043, Failures: 0, Errors: 0, Skipped: 0
+[INFO] 
+[INFO] 
+[INFO] --- maven-surefire-report-plugin:2.21.0:report-only (generate-test-report) @ hibernate-validator-tck-runner ---
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/jakartaee-certification/5.26.0/tck-results-web-5.26.0.adoc:56]: This and the following line can be removed

File: docs/modules/ROOT/pages/jakartaee-certification/5.26.0/tck-results-web-5.26.0.adoc [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+
+```
+ [mvn.test] [INFO] Tests run: 1655, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1,500.41 s - in TestSuite
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] Results:
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] Tests run: 1655, Failures: 0, Errors: 0, Skipped: 0
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] --- maven-surefire-report-plugin:2.19.1:report-only (generate-test-report) @ weld-payara-runner-tck11 ---
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/jakartaee-certification/5.26.0/tck-results-web-5.26.0.adoc:71]: This and the following line can be removed

File: docs/modules/ROOT/pages/jakartaee-certification/5.26.0/tck-results-full-5.26.0.adoc [hunk 3]
```diff
@@ -0,0 +1,756 @@
...
+
+```
+[INFO] Tests run: 1043, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 658.032 s - in TestSuite
+[INFO] 
+[INFO] Results:
+[INFO] 
+[INFO] Tests run: 1043, Failures: 0, Errors: 0, Skipped: 0
+[INFO] 
+[INFO] 
+[INFO] --- maven-surefire-report-plugin:2.21.0:report-only (generate-test-report) @ hibernate-validator-tck-runner ---
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/jakartaee-certification/5.26.0/tck-results-full-5.26.0.adoc:92]: This and the following line can be removed

File: docs/modules/ROOT/pages/jakartaee-certification/5.26.0/tck-results-full-5.26.0.adoc [hunk 4]
```diff
@@ -0,0 +1,756 @@
...
+
+```
+ [mvn.test] [INFO] Tests run: 1805, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1,800.088 s - in TestSuite
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] Results:
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] Tests run: 1805, Failures: 0, Errors: 0, Skipped: 0
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] 
+ [mvn.test] [INFO] --- maven-surefire-report-plugin:2.19.1:report-only (generate-test-report) @ weld-payara-runner-tck11 ---
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/jakartaee-certification/5.26.0/tck-results-full-5.26.0.adoc:107]: This and the following line can be removed

File: docs/modules/ROOT/pages/jakartaee-certification/5.26.0/tck-results-full-5.26.0.adoc [hunk 5]
```diff
@@ -0,0 +1,756 @@
...
+   [runcts] OUT => [javatest.batch] Completed running 30 tests.
+   [runcts] OUT => [javatest.batch] Number of Tests Passed      = 30
+   [runcts] OUT => [javatest.batch] Number of Tests Failed      = 0
+   [runcts] OUT => [javatest.batch] Number of Tests with Errors = 0
+```
+
+### beanvalidation
+
+```
+[INFO] Tests run: 1043, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 658.032 s - in TestSuite
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/jakartaee-certification/5.26.0/tck-results-full-5.26.0.adoc:85]: Missing indentation

File: docs/modules/ROOT/pages/jakartaee-certification/5.26.0/tck-results-web-5.26.0.adoc [hunk 6]
```diff
@@ -0,0 +1,397 @@
...
+- Summary of the information for the certification environment:
++
+Apache Derby, Linux, Ubuntu 18.04.3 +
+
+== Test results
+
+### beanvalidation
+
+```
+[INFO] Tests run: 1043, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 632.077 s - in TestSuite
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/jakartaee-certification/5.26.0/tck-results-web-5.26.0.adoc:49]: Missing indentation

**Review by rdebusscher — CHANGES_REQUESTED** [review]

**Comment by lprimak** [comment]
@rdebusscher this is a paste of a generated file. Do you really want me to make these changes?  I don’t think that’s sustainable going forward and not documented anywhere. 
I am just following what’s on confluence which is very involved as it is.

**Comment by rdebusscher** [comment]
@lprimak I compared it to the TCK result presentation of the previous releases.  We can do this also in a separate PR.

**Review by rdebusscher — APPROVED** [review]
Content OK, Small layout improvements later

**Review by Pandrex247 — APPROVED** [review]
Aside from Rudy's comments LGTM

**Pull request merged**